### DISCUSSION TITLE
python/docs/introduction/real-world-project

### DISCUSSION THREAD

**Discussion by giscus** [apify/crawlee-python#281] [Comments]
# python/docs/introduction/real-world-project

Hey, guys, you know, it's cool that we can scrape the `` elements of web pages, but that's not very useful. Can we finally scrape some real data and save it somewhere in a machine-readable format? Because that's why I started reading this tutorial in the first place!

https://crawlee.dev/python/docs/introduction/real-world-project

**Comment by drdollar** [comment]
Very well written this is, I will dive deeper in upcoming days as I am looking for a solution like yours for my project. Thank you!
